src/std/array.ab

```diff
@@ -23,6 +23,50 @@ pub fun array_search(array, value): [Num] {
 
 /// Checks if a value is in the array.
 pub fun includes(array, value) {
-    let result = array_first_index(array, value)
+    let result = array_index(array, value)
     return result >= 0
 }
+
+/// Returns the first element in the array
+pub fun first(array) {
```

Comment:
Should we also call these `array_first()` and `array_last()` while we're at it?

Reply:
The major argument for more verbose standard library function names is that we're unlikely to want to reuse them for builtins (a case in which my annotation idea would not help).  With this in mind, we may want a separate discussion on renaming the wider set of standard library functions.

Reply:
With this in mind, we may want a separate discussion on renaming the wider set of standard library functions, before we release 0.4.0.  I will create a discussion channel for this on Discord.